doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -4530,3 +4531,47 @@ A curl example using an ``ID``
   curl -X POST -H 'Content-Type:application/json' -d "$JSON" $SERVER_URL/api/admin/feedback
 
 Note that this call could be useful in coordinating with dataset authors (assuming they are also contacts) as an alternative/addition to the functionality provided by :ref:`return-a-dataset`.
+
+
+MyData
+------
+
+The MyData API is used to get a list of just the datasets, dataverses or datafiles an authenticated user can edit.
+
+MyData API
+~~~~~~~~~~
+
```

Comment:
```suggestion
```

We can probably remove this second level. Here's how it looks at https://dataverse-guide--9596.org.readthedocs.build/en/9596/api/native-api.html right now:

<img width="313" alt="Screenshot 2023-06-01 at 5 41 22 PM" src="https://github.com/IQSS/dataverse/assets/21006/4f565679-ec7b-4650-a04b-e7b8383d2b27">